tofu/rancher/cluster/main.tf

```
@@ -0,0 +1,177 @@
+provider "rancher2" {
+  api_url   = var.fqdn
+  token_key = var.api_key
+  insecure  = var.insecure
+}
+
+resource "random_string" "suffix" {
+  length  = 3
+  upper   = false
+  special = false
+}
+
+
+resource "rancher2_cloud_credential" "rancher2_cloud_credential" {
```

```suggestion
module "cloudcredential" {
  source = "../cloudcredential"
  ...
}
```
Can we use a "module" block instead of copy/pasting the `cloudcredential` module here?